examples/apps/ai-collab/CHANGELOG.md

```diff
@@ -1 +1,3 @@
 # @fluid-example/ai-collab
+
+## 2.4.0
```

Comment:
Not a big deal that this ended up empty, I suspect it's related to the fact that it didn't exist before. But in case we want to look into the logic and say something like "New package" in these cases.